lang/cs/Org.Apache.REEF.Common/Runtime/Evaluator/Context/RootContextLauncher.cs

```diff
@@ -39,17 +40,24 @@ internal sealed class RootContextLauncher
         private ISet<object> _services;
         private ContextRuntime _rootContext;
 
-        public RootContextLauncher(string id, IConfiguration contextConfiguration,
-            Optional<IConfiguration> rootServiceConfig, Optional<IConfiguration> rootTaskConfig, IHeartBeatManager heartbeatManager)
+        public RootContextLauncher(
```

Comment:
There is only one RootContextLauncher instance, right? Why we didn't use inject for it?